src/syrupy/assertion.py

```diff
@@ -62,8 +62,8 @@ def with_class(
             session=self._session,
         )
 
-    def assert_match(self, data) -> bool:
-        return self._assert(data)
+    def assert_match(self, data):
+        assert self == data
```

Comment:
Does pytest report the failure in syrupy/assertion.py though or in the original test?

Reply:
The failure is in the `assertion.py` file but the full trace shows the tests it fails in.
```sh
============================== FAILURES ===============================
_______________________ test_multiple_snapshots _______________________

snapshot = <SnapshotAssertion (2)>

    def test_multiple_snapshots(snapshot):
        assert "First." == snapshot
>       snapshot.assert_match("Second")

tests/test_snapshots.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <SnapshotAssertion (2)>, data = 'Second'

    def assert_match(self, data):
>       assert self == data
E       assert snapshot == Second
E         - Second
E         + Second.

src/syrupy/assertion.py:66: AssertionError
===================== 1 failed, 8 passed in 0.28s =====================
```

Reply:
Same behaviour with snapshottest at the moment